crypto/crypto/src/hash.rs

```diff
@@ -506,6 +516,20 @@ macro_rules! define_hasher {
         }
 
         impl CryptoHasher for $hasher_type {
+            fn seed() -> &'static [u8;32] {
+                if let Some(seed_value) = $seed_name.get() {
+                    seed_value
+                } else {
+                    let salt: Vec<u8> = [LIBRA_HASH_PREFIX, $salt].concat();
```

Comment:
Although we are in the same file this time, I would still call into `Default::make_seed` or something like that.